### ISSUE TITLE
Create npm package with the correct name under the correct org

### ISSUE THREAD

**Issue by plocket** [SuffolkLITLab/ALKiln#393]
Right now, it's called `docassemble-cucumber`. We need `alkiln`, I think. If Suffolk is interested, we also need to find out if Suffolk has a npm org and put these releases under that.

**Comment by plocket** [comment]
I think creating an npm org and project needs your participation, @colarusso

**Comment by colarusso** [comment]
FWIW, @nonprofittechy is an Owner as well. That being said, what is it exactly that is needed?

**Comment by plocket** [comment]
Packages use this framework by pulling it from the npm package registry (https://npmjs.com/). Right now, it's owned by me. I think folks from Suffolk should probably have the ability to publish, deprecate, and generally manage this framework. That means that Suffolk needs to have an npmjs organization (https://docs.npmjs.com/organizations) and create a package there, probably with the name alkiln, but it can be whatever you want.

If you can set us up, we can work on migrating to it. Let me know if you need more information.

@colarusso , @nonprofittechy

**Comment by plocket** [comment]
Want to double check scoped package decision. From the docs, another note about scoped repos:

> Note: You cannot transfer a scoped package to another user account or organization, because a package's scope is the user account or organization name. You will need to create a new package in the new scope.

---

Steps to transfer:

**Docs for transferring packages:** https://docs.npmjs.com/transferring-a-package-from-a-user-account-to-another-user-account

**Deprecate the old name and make a new name:** See https://docs.npmjs.com/cli/v8/commands/npm-deprecate or https://github.com/tiaanduplessis/pkg-rename/blob/master/index.js. Use @suffolklitlab/alkiln as the new repo name.

**Comment by colarusso** [comment]
I think our former call still stands. We'll put it under our org because that seems like the best form so as not to crowd out the namespace for all.

**Comment by plocket** [comment]
I've created the project and I'm all set with your part in this, colarusso. You can unwatch this issue if you want.

Next steps: we just need to go through and update the projects we're using. It's actually a convenient time. We can convert them to the new action and then they won't have to deal with the name of the package at all. I'd love a hand with this. Make a PR on the relevant repos in order of priority:

Live forms:
- [x] https://github.com/SuffolkLITLab/docassemble-AssemblyLine
- [ ] https://github.com/SuffolkLITLab/docassemble-Interpreternotice
- [ ] https://github.com/SuffolkLITLab/docassemble-MA209AProtectiveOrder
- [ ] https://github.com/SuffolkLITLab/docassemble-209aPlaintiffMotionToModify
- [ ] https://github.com/SuffolkLITLab/docassemble-MAHousingTRO
- [ ] https://github.com/GBLS/docassemble-MAEvictionDefense
- [ ] [need permissions to make new branches. will have to use channels] https://github.com/codeforboston/docassemble-juvenilesealing

Soon to be live:
- [ ] https://github.com/SuffolkLITLab/docassemble-AuthorizeFoodBenefitsRepresentative
- [ ] https://github.com/SuffolkLITLab/docassemble-AppealsBriefTemplate
- [ ] https://github.com/MassLegalHelp/docassemble-HelpForChildSupportObligors

Deprecated or stable:
- [ ] https://github.com/SuffolkLITLab/docassemble-MAVirtualCourt
- [ ] https://github.com/SuffolkLITLab/docassemble-MAEvictionMoratorium

Used for testing:
- [x] https://github.plocket/docassemble-CompositeActionsTest

Dependents, but unecessary:
- [ ] https://github.com/SuffolkLITLab/docassemble-PDFConcatTests
- [ ] https://github.plocket/tests-github-action-push
- [ ] https://github.SuffolkLITLab/docassemble-IntegratedTesting

What we need to do to each repo is first make sure the tests can pass with version 3 of alkiln:
- Clone the repo
- Checkout a new branch. Let's call it `alkiln_update` to be consistent.
- In /package.json, edit `"docassemble-cucumber": "whatever they have in here",` to `"docassemble-cucumber": "^3.0.0",`. Yes, I'm aware that isn't the right name, but it's equivalent and it does the least to rock the boat at the moment.
- Run `npm install`
- Create the file /.env. Use the /.env_example file to put in the data from your own docassemble account on the server.
- Try to convert v1 tests to v3 using the example below and the [kiln documentation](https://suffolklitlab.org/docassemble-AssemblyLine-documentation/docs/automated_integrated_testing/).

For v1 story table, this:
```
    | var | value | checked |
    | a_plaintiff_s_motion_to_modify_qc_v20009 | True | true |
    | signature | users[0].signature |  |
    | trial_court | all_courts[0] | true |
```

will turn into something like:
```
    | var | value | trigger |
    | a_plaintiff_s_motion_to_modify_qc_v20009 | True |  |
    | users[0].signature |  | users[0].signature |
    | trial_court.all_courts[0] | True |  |
```

No guarantees that it'll work.

For v2 story table, most things should work fine.

After doing your best, do
- Run `npm run local`
- If the tests pass, you should be good. If they don't and you can tell why, fiddle with it. If you can't tell, message me.
- To speed up iteration of testing, you can use `npm run setup` once, `npm run cucumber` repeatedly to run the tests themselves, then `npm run takedown` when you're done.

Once the tests pass, transition the repo to our new action-only flow. Use [Appeals Brief Template's PR](https://github.com/SuffolkLITLab/docassemble-AppealsBriefTemplate/pull/37) as an example of the final product. Some of these files will have been added by you when you set up to run the tests, so remember to delete all of them.
- Delete /package-lock.json
- Delete /node_modules
- Delete /package.json
- Delete /.env
- Delete /.env_example
- Delete /.gitignore
- Edit their /.github/workflows/run_form_tests.yml to [look like this](https://github.com/plocket/docassemble-CompositeActionsTest/blob/441_action_dev/.github/workflows/run_form_tests.yml)
- Push that and see if the tests pass.
- Make a PR. I've used:

Title: Test with alkiln v3 action-controlled workflow only
Body: This uses only the workflow file to trigger ALKiln tests. Removed package.json and other npm files and files for local development as the ALKiln will manage these now. References the ALKiln v3 branch to keep up with updates to that branch.

- Get a review? Just merge?
- Celebrate!

**Comment by plocket** [comment]
Closed by #527
